<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StylePage</class>
 <widget class="QWizardPage" name="StylePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Style:</string>
        </property>
        <property name="buddy">
         <cstring>styleCombo</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="styleCombo">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>Circle (clipped)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Square (clipped)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ellipse</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Rectangle</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Thumbnail size:</string>
        </property>
        <property name="buddy">
         <cstring>thumbSize</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="thumbSize">
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="value">
         <number>128</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Anchor size:</string>
        </property>
        <property name="buddy">
         <cstring>anchorSize</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="anchorSize">
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="value">
         <number>32</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Border color:</string>
        </property>
        <property name="buddy">
         <cstring>color1</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="ColorEditor" name="color1"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Outline color:</string>
        </property>
        <property name="buddy">
         <cstring>color2</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="ColorEditor" name="color2"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Border thickness:</string>
        </property>
        <property name="buddy">
         <cstring>lineweight</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QSpinBox" name="lineweight">
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Text:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="2">
       <widget class="QPlainTextEdit" name="text">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="2">
       <widget class="QFrame" name="frame_2">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QFontComboBox" name="Font"/>
         </item>
         <item>
          <widget class="FontSizeCombo" name="fontSize">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>0</height>
            </size>
           </property>
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="GraphicsView" name="view2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>250</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>graphicsview.h</header>
  </customwidget>
  <customwidget>
   <class>ColorEditor</class>
   <extends>QComboBox</extends>
   <header>widgets.h</header>
  </customwidget>
  <customwidget>
   <class>FontSizeCombo</class>
   <extends>QComboBox</extends>
   <header>widgets.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
